communities.json

```diff
@@ -115,7 +113,8 @@
     "https://falterturm.freifunk-vec.de/meshviewer.json"
   ],
   "Königswinter": [
```

Comment:
Maybe Königswinter should be renamed to DiVoNet. From the looks of it that seems to be a construct that is somewhat regional but if you zoom the card out you get quite a few nodes (freifunk-thw.de) outside of that area ...

But idk, can also stay like this for now.

Reply:
As the old map at https://ffkwsn03.freifunk-koenigswinter.de/data/meshviewer.json did not contain any nodes besides the single supernode, I renamed the community to Divonet in my latest Commit.
Thanks!